chat-widget/src/common/telemetry/definitions/Payload.ts

```diff
@@ -98,6 +98,13 @@ export interface CallingTelemetryData extends BaseTelemetryData {
     Description?: string;
 }
 
+export interface FacadeChatSDKTelemetryData extends BaseTelemetryData {
+    Event?: string;
```

Comment:
Do all these types need to be optional?

Reply:
is just the pattern, it really doesnt affect